pkg/network/network.go

```diff
@@ -45,6 +45,11 @@ const (
 	// IstioIngressClassName value for specifying knative's Istio
 	// ClusterIngress reconciler.
 	IstioIngressClassName = "istio.ingress.networking.knative.dev"
+
+	// RouteTemplateKey is the name of the configuration entry that
+	// specifies the golang template string to use to construct the
+	// Knative service's DNS name.
+	RouteTemplateKey = "routeTemplate"
```

Comment:
Should this be "domainTemplate"?

Reply:
either works for me. I'll change it.

Reply:
Switched back to RouteTemplate since the value is more than just the domain, it's the full URL/route.

Reply:
I'd still say it's "domain". You're generating the (sub)-domain for the route. Point in case the function using this is called `routeDomain`

Reply:
But the string/URL generated has the top-most value, the function name, and we don't expect it to act as a domain for other things, do we?

Reply:
I can rename `routeDomain` if you want :-)  I think it's misleading